setup.cfg

```diff
@@ -36,7 +36,7 @@ py_modules = nbqa
 install_requires =
     ipython>=7.8.0
     tokenize-rt>=3.2.0
-    tomli<2.0.0
+    tomli
```

Comment:
I should never have accepted this upper-bound, but whatever, too late now, all I can do is remove it

Reply:
Better upper bounds and testing that it works than stuff that breaks all of the sudden all over the place 😝

Reply:
dunno, the risk is that then `pip` doesn't let people install things because of unnecessary constraints their packages have specified